cli/tests/unit/webcrypto_test.ts

```diff
@@ -608,6 +608,109 @@ Deno.test(async function testAesCbcEncryptDecrypt() {
   assertEquals(new Uint8Array(decrypted), new Uint8Array([1, 2, 3, 4, 5, 6]));
 });
 
+Deno.test(async function testAesCtrEncryptDecrypt() {
+  async function aesCtrRoundTrip(
+    key: CryptoKey,
+    counter: Uint8Array,
+    length: number,
+    plainText: Uint8Array,
+  ) {
+    const cipherText = await crypto.subtle.encrypt(
+      {
+        name: "AES-CTR",
+        counter,
+        length,
+      },
+      key,
+      plainText,
+    );
+
+    assert(cipherText instanceof ArrayBuffer);
+    assertEquals(cipherText.byteLength, plainText.byteLength);
```

Comment:
Excess of caution: consider adding a check that verifies cipherText != plainText, otherwise you'd never know it if encrypt and decrypt are identity functions. Likewise around line 642.

Reply:
done @ 628. `cryptoText != plainText`. 
Not required @642 since already checking if `decryptedText == plainText`, and `cryptoText != plainText`